src/components/Accordion/AccordionItem.tsx

```diff
@@ -66,7 +66,7 @@ export const AccordionItem: FC<AccordionItemProps> = ({
       >
         <div>{title}</div>
 
-        <Icon icon={isOpen ? 'icon-angle-down' : 'icon-angle-up'} />
+        <Icon icon={isOpen ? 'icon-angle-up' : 'icon-angle-down'} />
```

Comment:
po tylu latach się odwidziało xd

Reply:
bo ten accordion z ui-kita to był tylko u was na tej checkliście edytorowej używany XD